test/metabase/query_processor/middleware/parameters/mbql_test.clj

```diff
@@ -94,10 +94,6 @@
 (deftest ^:parallel date-ranges-e2e-test
   (mt/test-drivers (params-test-drivers)
     (testing "check that date ranges work correctly"
-      ;; Prevent an issue with Snowflake were a previous connection's report-timezone setting can affect this test's
-      ;; results
-      (when (= :snowflake driver/*driver*)
-        (driver/notify-database-updated driver/*driver* (mt/id)))
```

Comment:
I'd like to find out if this is resolved by this PR. If not it will be reverted.

Reply:
Leaving just TODO in for now.